### PR TITLE
Document ES6 mode for :uglify_js filter

### PR DESCRIPTION
**Pull request by da2x**
This should hopefully save someone some time.

### CODE REVIEW

File: content/doc/reference/filters.dmark [hunk 1]
```diff
@@ -556,10 +556,10 @@ title: "Filters"
   #listing[lang=ruby]
     filter :uglify_js
 
-  #p This filter takes options and passes them on to %code{Uglifier#initialize}. For example, the following will disable name mangling:
+  #p This filter takes options and passes them on to %code{Uglifier#initialize}. For example, the following will disable name mangling and enable support for Harmoney mode (ES6/ES2015+):
```
Comment (ddfreyne) [inline content/doc/reference/filters.dmark]: "Harmoney" should be "Harmony"

File: content/doc/reference/filters.dmark [hunk 2]
```diff
@@ -556,10 +556,10 @@ title: "Filters"
...
 
   #listing[lang=ruby]
-    filter :uglify_js, mangle: false
+    filter :uglify_js, mangle: false, :harmony => true
```
Comment (ddfreyne) [inline content/doc/reference/filters.dmark]: Can you change this to

```
harmony: true
```

(i.e. Ruby 1.9 hash syntax)

File: content/doc/reference/filters.dmark [hunk 3]
```diff
@@ -556,10 +556,10 @@ title: "Filters"
   #listing[lang=ruby]
     filter :uglify_js
 
-  #p This filter takes options and passes them on to %code{Uglifier#initialize}. For example, the following will disable name mangling:
+  #p This filter takes options and passes them on to %code{Uglifier#initialize}. For example, the following will disable name mangling and enable support for Harmony mode (ES6/ES2015+):
 
   #listing[lang=ruby]
-    filter :uglify_js, mangle: false
+    filter :uglify_js, mangle: false, harmony => true
```
Comment (ddfreyne) [inline content/doc/reference/filters.dmark]:
```suggestion
    filter :uglify_js, mangle: false, harmony: true
```

**Comment by da2x** [comment]
Yes, of course. Sorry about the mess with this one. I originally wrote this through GitHub’s web interface on my phone while waiting for a bus. :upside_down_face:

**Comment by ddfreyne** [comment]
All good, thanks!

**Pull request merged**